charts/cloudzero-agent/README.md

```diff
@@ -121,29 +121,24 @@ Please see the [sizing guide](./docs/sizing-guide.md) in the docs directory.
 
 ### Metric Exporters
```

Comment:
@bdrennz one of the things that I was a bit confused about when testing this chart is what to do when using an existing KSM, and it seems like customers get tripped up too. maybe it would be nice to have a section that more clearly spells out the options for KSM, and how to set the service address for KSM if providing an existing service

Reply:
Totally, that's a good idea. I'll add some documentation around it 👍